packages/near-snap/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@near-wallet-selector/near-snap",
+  "version": "v7.3.0",
+  "peerDependencies": {
+    "near-api-js": "^0.44.2 || ^1.0.0"
+  }
+}
```

Comment:
We have recently made some updates in the `package.json` of each package to add some additional meta data.
Can you please align this with the other packages?